app/views/posts/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1><%= post.title %></h1>
+<% if post&.featured? %>
```

Comment:
I don't think you need the pretzel here as line 1 will error if `post` is `nil`.